python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -67,8 +67,9 @@ def _process_cli_config(cli_config: AIConfigCLIConfig) -> Result[bool, str]:
 def _run_frontend_server_background() -> Result[list[subprocess.Popen[bytes]], str]:
     LOGGER.info("Running frontend server in background")
     try:
-        p1 = subprocess.Popen(["yarn"], cwd="src/aiconfig/editor/client")
-        p2 = subprocess.Popen(["yarn", "start"], cwd="src/aiconfig/editor/client")
+        p1 = subprocess.Popen(["yarn"], cwd="python/src/aiconfig/editor/client")
+        p2 = subprocess.Popen(["yarn", "start"], cwd="python/src/aiconfig/editor/client", stdin=subprocess.PIPE)
```

Comment:
Add comment TODO to run from `aiconfig` root dir

Reply:
?